src/sentry/models/group.py

```diff
@@ -419,6 +425,14 @@ class Group(Model):
         ),
         db_index=True,
     )
+    substatus = BoundedIntegerField(
```

Comment:
Just want to confirm - in the title you've called it `sub_status`, but here it's `substatus`. Just want to make sure we know what we want to call this

Reply:
Yeah, was going back and forth on this since sub_status is maybe the more proper way but I think substatus is just easier to use.